appsec/src/helper/remote_config/client.cpp

```diff
@@ -207,6 +231,12 @@ bool client::process_response(std::set<config> new_configs)
             listener->commit();
         } catch (const std::exception &e) {
             SPDLOG_ERROR("Failed to commit listener: {}", e.what());
+            if (msubmitter_) {
+                msubmitter_->submit_log(log_level::Error,
+                    "remote_config_client_commit_error",
+                    fmt::format("Failed to commit listener: {}", e.what()),
+                    std::nullopt, std::nullopt, false);
+            }
```

Comment:
The logs don't seem to comply with the RFC requirements, which require sending diagnostics (albeit other errors are fine as well)

Reply:
this should be fixed in https://github.com/DataDog/dd-trace-php/pull/3236/commits/ce36b573fe1ca54c429ccccbb6fbfd194851c9ff